go/libraries/doltcore/schema/alterschema/addcolumn_test.go

```diff
@@ -116,7 +115,7 @@ func TestAddColumnToTable(t *testing.T) {
 			defaultVal: mustStringToColumnDefault(`("default")`),
 			expectedSchema: dtestutils.AddColumnToSchema(dtestutils.TypedSchema,
 				schemaNewColumn("newCol", dtestutils.NextTag, types.StringKind, false, `("default")`, schema.NotNullConstraint{})),
-			expectedRows: dtestutils.AddColToRows(t, dtestutils.TypedRows, dtestutils.NextTag, types.String("default")),
+			expectedRows: dtestutils.AddColToRows(t, dtestutils.TypedRows, dtestutils.NextTag, types.NullValue),
```

Comment:
This is fine to check in, but you need to fast followup with additional tests, probably in `DoltScripts`. This code used to exercise the "fill in the default value of existing rows with new column" logic, and it doesn't do that anymore.

So 1) reproduce these tests as SQL scripts, and 2) pare these down so they're just unit tests for the AddColumnToTable method